### DISCUSSION TITLE
同学们输出的文章

### DISCUSSION THREAD

**Discussion by cuixiaorui** [cuixiaorui/mini-vue#57] [General]
收集同学们基于课程产出的文章

**Comment by You-5C24** [comment]
[专栏：吃透mini-vue](https://juejin.cn/column/7071633886781898782)

**Comment by Leiloloaa** [comment]
[10张图、10分钟，包你学会 Vue3 的“左右互博之术”](https://juejin.cn/post/7053996176051273765)

**Reply by Leiloloaa** [reply]
[7张图、20分钟，包你学会 Vue3 最优化计算 vdom](https://juejin.cn/post/7054055241704013831)

**Reply by Leiloloaa** [reply]
[Vue3 源码内参<一>手写mini前置准备](https://juejin.cn/post/7028471826627952671)

**Reply by Leiloloaa** [reply]
[Vue3 源码内参<二>Reactivity核心](https://juejin.cn/post/7029201805036748836)

**Comment by nikolausliu** [comment]
- [B站手写mini-vue教程笔记](https://juejin.cn/post/7068532458349658119)
- [手撕mini-vue源码——环境搭建](https://juejin.cn/post/7068864334268465182)
- [手撕mini-vue源码——实现effect & reactive & track & trigger](https://juejin.cn/post/7069984812013977630)

**Comment by chenfan0** [comment]
[手写mini-vue3:基础环境的搭建](https://juejin.cn/post/7065504184077910052)
[手写mini-vue3：实现reactivity模块中的-reactive-effect](https://juejin.cn/post/7065630386931367944)
[手写mini-vue3：实现reactivity模块中的-ref](https://juejin.cn/post/7065896093870981133)
[手写mini-vue3：实现reactivity模块中的-computed](https://juejin.cn/post/7066270655842353166)
[手写mini-vue3：实现初始化component的主流程](https://juejin.cn/post/7067181816670584846)

**Comment by wenzheng-yi** [comment]
[Vue3 响应式 reactive、ref、effect 流程解析](https://juejin.cn/post/7043265644594200613)

**Comment by qinran0423** [comment]
#### 掘金
[mini-vue之reactivity库的实现](https://juejin.cn/post/7045921581788299278)
#### 读源码我的思考
[源码调试](https://github.com/qinran0423/vue3-source-study/blob/main/doc/%E6%BA%90%E7%A0%81%E8%B0%83%E8%AF%95.md)
[初始化流程](https://github.com/qinran0423/vue3-source-study/blob/main/doc/%E5%88%9D%E5%A7%8B%E5%8C%96%E6%B5%81%E7%A8%8B.md)
[首次 patch](https://github.com/qinran0423/vue3-source-study/blob/main/doc/%E9%A6%96%E6%AC%A1patch.md)
[属性初始化](https://github.com/qinran0423/vue3-source-study/blob/main/doc/%E5%B1%9E%E6%80%A7%E5%88%9D%E5%A7%8B%E5%8C%96.md)
[slot](https://github.com/qinran0423/vue3-source-study/blob/main/doc/slot.md)
[响应式](https://github.com/qinran0423/vue3-source-study/blob/main/doc/%E5%93%8D%E5%BA%94%E5%BC%8F.md)
[异步更新](https://github.com/qinran0423/vue3-source-study/blob/main/doc/nexTick.md)
[effect](https://github.com/qinran0423/vue3-source-study/blob/main/doc/effect.md)
[readonly](https://github.com/qinran0423/vue3-source-study/blob/main/doc/readonly.md)
[isReactive和isReadonly](https://github.com/qinran0423/vue3-source-study/blob/main/doc/isReactive%E5%92%8CisReadonly.md)
[ref](https://github.com/qinran0423/vue3-source-study/blob/main/doc/ref.md)
[isRef和unRef](https://github.com/qinran0423/vue3-source-study/blob/main/doc/isRef%E5%92%8CunRef.md)
[computed](https://github.com/qinran0423/vue3-source-study/blob/main/doc/computed.md)

**Comment by Merlin218** [comment]
- [「手写vue3系列」响应式实现](https://juejin.cn/post/7028613132339642382)
- [「手写vue3系列」一个Vue应用的构建过程](https://juejin.cn/post/7030238460476653582)

**Comment by Stan9726** [comment]
欢迎在掘金关注[我](https://juejin.cn/user/158789396594765)或者关注我的[手写 Mini-Vue3 专栏](https://juejin.cn/column/7028767798591815710)，如果觉得不错还请点赞👍支持一下，专栏文章列表：

1. [了解 Vue3 源码结构 + 初始化项目](https://juejin.cn/post/7029145067755421726)
2. [实现 reactivity - reactive 和 effect](https://juejin.cn/post/7028771190416080933)
3. [实现 reactivity - readonly、shallowReactive 和 shallowReadonly](https://juejin.cn/post/7029147690906697765)
4. [实现 reactivity - ref 和 computed](https://juejin.cn/post/7029147829134164004)
5. [实现 runtime-core - Component 和 Element 初始化，以及组件代理对象、shapeFlags 和 注册事件](https://juejin.cn/editor/drafts/7037021594861961229)

**Comment by alexzhang1030** [comment]
[mini-vue 笔记](https://github.com/zx-projects/mini-vue-docs)

**Comment by littlewhitehhh** [comment]
mini-vue配合《Vuejs设计和实现》学习笔记
 - [Vue3源码学习（1）--框架设计概览](https://juejin.cn/post/7074111898894991390/) 
 - [Vue3 源码学习（2）--响应式系统（1）](https://juejin.cn/post/7074496267061035038/)
 - [Vue3 源码学习（3）--响应式系统（2）](https://juejin.cn/post/7074847535621210126/)
 - [Vue3 源码学习（4）--响应式系统（3）](https://juejin.cn/post/7075139625592815624/)
 - [Vue3 源码学习（5）--runtime-core（1）](https://juejin.cn/post/7079687116841549855)

**Comment by zhangchongyang01** [comment]
专栏：mini-vue-learning(https://juejin.cn/column/7087568611899883551)

**Comment by HongxuanG** [comment]
[专题：Vue3核心原理代码解构](https://juejin.cn/column/7089244418703622175)

**Comment by DanmoSAMA** [comment]
[掘金专栏 - 手写mini-vue](https://juejin.cn/column/7260058744485724197)
